### ISSUE TITLE
rtags-rdm-includes crash rdm start

### ISSUE THREAD

**Issue by SimeonEhrig** [Andersbakken/rtags#1353]
**Describe the bug**
I want to set the include path for the std C++ lib at start of rdm. So I added the following line to my init file `(setq rtags-rdm-includes "/usr/include/c++/5.5.0")`. I use the `c++-mode-hook` that starts the rdm server when I open a `.cpp` file. Without `rtags-rdm-includes` the rdm server is starts and I can use rtags commands. If I set the variable, I get the error message `File mode specification error: (wrong-type-argument sequencep 47)` in my minibuffer and rdm is not started.

**To Reproduce**
1. Add to your init file:
```lisp
;; change the path depending on your system 
(setq rtags-rdm-includes "/usr/include/c++/5.5.0")
(add-hook 'c++-mode-hook 'rtags-start-process-unless-running)
```
2. Reload the init file or restart emacs
3. Open a `.cpp` file
4. Look at the minibuffer or Message buffer

**Expected behavior**
The rdm server should start when I open a `.cpp` file and the include path should be set.

**Environment (please complete the following information):**
 - OS/Version: Ubuntu 16.04.6
 - RTags(rdm,rc): 2.34
 - Emacs: 26.3
 - LLVM/Clang: 8.0

**Additional context**
At the moment, I'm using `(setq rtags-process-flags "-I/usr/include/c++/5.5.0)` as workaround.

**Comment by cslux** [comment]
The `:type` of `rtags-rdm-includes` is wrong. `rtags-rdm-includes` MUST be a list of strings not a string.

```
(setq rtags-rdm-include '("/usr/include/c++/5.5.0"))
```

**Comment by cslux** [comment]
@SimeonEhrig Fixed via ca131c1.
